numpy/_core/src/umath/ufunc_object.c

```diff
@@ -6434,6 +6444,20 @@ ufunc_get_doc(PyUFuncObject *ufunc, void *NPY_UNUSED(ignored))
     return doc;
 }
 
+static int
+ufunc_set_doc(PyUFuncObject *ufunc, PyObject *doc, void *NPY_UNUSED(ignored))
+{
+    if (doc == NULL) {
+        int result = PyDict_Contains(ufunc->dict, npy_interned_str.__doc__);
+        if (result == 1) {
+            return PyDict_DelItem(ufunc->dict, npy_interned_str.__doc__);
+        } else {
+            return result;
+        }
```

Comment:
This seems weird/wrong, why not just call `PyDict_DelItem` and that's that!?

Reply:
Right, I can just call `PyDict_DelItem`, done!